SYCL/Basic/cgeist/parallel_for.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -Xcgeist -gen-all-sycl-funcs %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// REQUIRES: linux
+// UNSUPPORTED: hip || cuda
+
+#include <sycl/sycl.hpp>
+using namespace sycl;
+#define N 8
```

Comment:
Can we have this defined as `static constexpr` instead?

Reply:
Sure, why is it preferred?